static/openapi/eds-spec.json

```diff
@@ -0,0 +1,399 @@
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/es/v1/metrics": {
+      "get": {
+        "summary": "Retrieve Metrics",
+        "description": "Retrieve metrics data within a specified time range.",
+        "tags": [
+          "Event Delivery Metrics"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "start",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The start date used for querying metrics data. Must be provided in RFC3339 date-time format.",
+            "example": "YYYY-MM-DDTHH:MM:SSZ"
+          },
+          {
+            "in": "query",
+            "name": "end",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The end date used for querying metrics data. Must be provided in RFC3339 date-time format.",
+            "example": "YYYY-MM-DDTHH:MM:SSZ"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the metrics data to retrieve."
+          },
+          {
+            "in": "query",
+            "name": "limit",
+            "required": false,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The maximum number of metrics to retrieve (optional)."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved metrics.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "domain_key": {
+                        "type": "string",
+                        "description": "The domain key."
+                      },
+                      "es_cpu_usage_second": {
+                        "type": "number",
+                        "description": "ES CPU usage per second."
+                      },
+                      "es_errors_per_second": {
+                        "type": "number",
+                        "description": "ES errors per second."
```

Comment:
Do you think we need to put ES in every one of these descriptions?
May it should be "Stream errors per second"?
I don't know what to do with these statistics.
Interesting that there is a lot of GDN info here (fabric/tenant).  With the users of this API know what to do with this information?  You might have to give more context for these values.

Reply:
Removed ES from all the stats description. I think one of the use of these statistics will be to display in in the GUI when it gets developed with charts.